### ISSUE TITLE
Update to React 16

### ISSUE THREAD

**Issue by rogovdm** [react-ga/react-ga#178]

**Comment by SimeonC** [comment]
We shouldn't have any problems running under React 16. Our React dependency is now optional so you should just be able to use React 16 as is. Let me know if anyone tries this out.

**Comment by ryank311** [comment]
It's optional but because it uses create-react-class@15.6.2, yarn actually pulls in the entire react 15x when the rest of our app is on 16x. I created a PR for this #211

**Comment by SimeonC** [comment]
#211 has been merged and versioned in 2.4.1.

**Comment by simonkberg** [comment]
Hi @SimeonC, thanks for trying to get this out. It seems however like the version was never published to npm; I can only find the release here on github while the latest published version is still 2.4.0.

```sh
$ npm info react-ga dist-tags.latest
2.4.0
```

**Comment by SimeonC** [comment]
Sorry about that, looks like I forgot to `npm publish`. Now it reports:
```
> npm info react-ga dist-tags.latest
2.4.1
```
